lib/developer_portal/app/views/developer_portal/user/show.html.liquid

```diff
@@ -11,20 +11,47 @@
       <div class="panel-body panel-footer">
         {% form 'user.personal_details', user, class: "form-horizontal" %}
           <fieldset>
+            <legend>
+              <span>User information</span>
+            </legend>
             {% for field in user.fields %}
               {% include 'field' with field %}
             {% endfor %}
           </fieldset>
 
           <fieldset>
+            <legend>
+              <span>Provide your current password</span>
+            </legend>
+            <div class="form-group {{ user.errors.current_password | error_class: 'has-error' }}">
+              <label for="user_current_password" class="col-md-4 control-label">
+                Current Password
+              </label>
+              <div class="col-md-6">
+                <input
+                  id="user_current_password"
+                  name="user[current_password]"
+                  type="password"
+                  class="form-control">
+                {{ user.errors.current_password | inline_errors: 'help-block' }}
+              </div>
+            </div>
```

Comment:
If user not `.using_password?`, will this be disabled / hidden? How does it work in that scenario?

Reply:
I believe it's not being handled, but I'm taking care of this.